html/php-pre-challenge2/index.php

```diff
@@ -3,7 +3,13 @@
 
 // 修正はここから
 for ($i = 0; $i < count($array); $i++) {
-
+    for ($j = 0; $j < count($array) - 1 - $i; $j++) { //比較する残りの値の数くりかえし
```

Comment:
```suggestion
    for ($j = 0; $j < (count($array) - 1 - $i); $j++) { //比較する残りの値の数くりかえし
```

比較を行うときに、条件が複雑（例：今回のように計算が入る）場合は、()などをつけて、比較の優先度を明示してあげると親切です。

今回は、この状態でも良かったですが、掛け算や論理演算子（ANDとかOR）など入ると複雑になり
予期せぬ不具合を起こしやすくするので。